python/whylogs/core/dataset_profile.py

```diff
@@ -129,6 +147,92 @@ def _do_track(
 
         raise NotImplementedError
 
+    def track_classification_metrics(
```

Comment:
this API is extremely error prone so I'm not sure we should even expose it here as a public API. Probably should add underscore to it at least

Reply:
Done.